src/sortStudents.ts

```diff
@@ -1,16 +1,62 @@
 
 export interface Student {
-  // describe Student interface
+  name:string;
+  surname:string;
+  age:number;
+  married:boolean;
+  grades:number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAverageGrade(grades:number[]):number {
+  return grades.reduce((previous:number, current:number) => previous
+  + current, 0) / grades.length;
+}
+
+export function sortStudents(
+  students:Student[],
+  sortBy:SortType,
+  order:SortOrder,
+):Student[] {
+  const sortedStudents = [...students];
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      return order === 'asc'
+        // eslint-disable-next-line max-len
+        ? sortedStudents.sort((first, second) => first[sortBy].localeCompare(second[sortBy]))
+        // eslint-disable-next-line max-len
+        : sortedStudents.sort((first, second) => second[sortBy].localeCompare(first[sortBy]));
```

Comment:
Fix your code style without disabling linter.
For example
```suggestion
        : sortedStudents.sort((first, second) => (
            second[sortBy].localeCompare(first[sortBy])
          ));
```